server/src/internalClusterTest/java/org/opensearch/action/admin/cluster/node/tasks/ConcurrentSearchTasksIT.java

```diff
@@ -109,7 +109,7 @@ public void testConcurrentSearchTaskTracking() {
 
             Map<Long, List<ThreadResourceInfo>> threadStats = getThreadStats(SearchAction.NAME + "[*]", taskInfo.getTaskId());
             // Concurrent search forks each slice of 5 segments to different thread
-            assertEquals((int) Math.ceil(getSegmentCount(INDEX_NAME) / 5.0), threadStats.size());
+            assertEquals((int) Math.ceil(getSegmentCount(INDEX_NAME) / 5.0) + 1, threadStats.size());
```

Comment:
@sohami as we know, in pre - Apache Lucene 9.8 the `Executor` was wrapped in `TaskExecutor` that invoked the last task on caller thread, this wrapping has been removed now.

Reply:
lgtm. Can we also add reference to [this issue](https://github.com/apache/lucene/issues/12498) in the comment